### DISCUSSION TITLE
/architecutre-concepts/request-lifecycle.html

### DISCUSSION THREAD

**Discussion by giscus** [goravel/goravel#365] [Announcements]
# /architecutre-concepts/request-lifecycle.html

[[toc]] Introduction The main.go file serves as the entry point for all requests in the Goravel application. It utilizes the bootstrap.Boot() function to initialize the framewor...

https://www.goravel.dev/architecutre-concepts/request-lifecycle.html

**Comment by raydenka21** [comment]
how to implement something like this https://laravel.com/docs/10.x/redis ?

**Reply by hwbrzzl** [reply]
We don't have a dependent Redis facade for now, you can find a Redis package and configure it in the framework first.
